functions/src/modules/gmail.functions.ts

```diff
@@ -0,0 +1,220 @@
+import * as functions from 'firebase-functions';
+import * as admin from 'firebase-admin';
+import * as Google from 'googleapis';
+import { Message } from 'firebase-functions/lib/providers/pubsub';
+import { processSQRDoneFromGmail } from './SQR.functions';
```

Comment:
I'd like to see something like this (may not be correct syntactically, just the idea):
```suggestion
import { processMarkingSubmissionAsDone } as SQR from './SQR.functions';
```
So that later we can call this function as `SQR.processMarkingSubmissionAsDone`.

Reply:
Only this `import * SQR from './SQR.functions'` works.

Reply:
I think it's okay.